### ISSUE TITLE
[BUG] open \\.\pipe\crc-admin-helper: Access is denied.

### ISSUE THREAD

**Issue by w4hf** [code-ready/crc#2455]
### General information

  * OS:  Windows
  * Hypervisor: Hyper-V
  * Did you run `crc setup` before starting it ? YES
  * Running CRC on: Laptop (Desktop)

## CRC version
```bash
CodeReady Containers version: 1.28.0+08de64bd
OpenShift version: 4.7.13 (embedded in executable)
```
  
## CRC status
```bash
DEBU CodeReady Containers version: 1.28.0+08de64bd
DEBU OpenShift version: 4.7.13 (embedded in executable)
DEBU Running 'crc status'
DEBU Checking file: C:\Users\HOMY\.crc\machines\crc\.crc-exist
DEBU Running '(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion" -Name ReleaseId).ReleaseId'
DEBU Running '(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").EditionID'
Machine does not exist. Use 'crc start' to create it
```

## CRC config
```bash
- consent-telemetry                     : yes
```
## CRC setup
```
INFO Checking if crc-admin-helper executable is cached
INFO Checking for obsolete admin-helper executable
INFO Checking if running on a supported CPU architecture
INFO Checking minimum RAM requirements
INFO Checking if running in a shell with administrator rights
INFO Checking Windows 10 release
INFO Checking Windows edition
INFO Checking if Hyper-V is installed and operational
INFO Checking if crc-users group exists
INFO Checking if current user is in Hyper-V group and crc-users group
INFO Checking if Hyper-V service is enabled
INFO Checking if the Hyper-V virtual switch exist
INFO Found Virtual Switch to use: Default Switch
INFO Checking if admin-helper daemon is installed
INFO Installing admin-helper daemon
INFO Will run as admin: install admin-helper daemon
INFO Checking if CRC bundle is extracted in '$HOME/.crc'
INFO Checking if C:\Users\HOMY\.crc\cache\crc_hyperv_4.7.13.crcbundle exists
INFO Extracting bundle from the CRC executable
INFO Ensuring directory C:\Users\HOMY\.crc\cache exists
INFO Extracting embedded bundle crc_hyperv_4.7.13.crcbundle to C:\Users\HOMY\.crc\cache
INFO Uncompressing crc_hyperv_4.7.13.crcbundle
crc.vhdx: 11.57 GiB / 11.57 GiB [-----------------------------------------------------------------------------] 100.00%
Your system is correctly setup for using CodeReady Containers, you can now run 'crc start' to start the OpenShift cluster
``` 

## Host Operating System
```bash
Host Name:                 HOMY
OS Name:                   Microsoft Windows 10 Professionnel
OS Version:                10.0.19042 N/A Build 19042
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
Registered Owner:          HOMY
Registered Organization:
Product ID:                00330-80000-00000-AA440
Original Install Date:     20/11/2020, 12:50:49
System Boot Time:          20/06/2021, 13:57:10
System Manufacturer:       Micro-Star International Co., Ltd.
System Model:              MS-7B51
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 158 Stepping 10 GenuineIntel ~3192 Mhz
BIOS Version:              American Megatrends Inc. 1.30, 12/11/2018
Windows Directory:         C:\WINDOWS
System Directory:          C:\WINDOWS\system32
Boot Device:               \Device\HarddiskVolume2
System Locale:             en-gb;English (United Kingdom)
Input Locale:              fr;French (France)
Time Zone:                 (UTC+01:00) Brussels, Copenhagen, Madrid, Paris
Total Physical Memory:     16,322 MB
Available Physical Memory: 11,274 MB
Virtual Memory: Max Size:  29,122 MB
Virtual Memory: Available: 21,183 MB
Virtual Memory: In Use:    7,939 MB
Page File Location(s):     C:\pagefile.sys
Domain:                    WORKGROUP
Logon Server:              \\HOMY
Hotfix(s):                 10 Hotfix(s) Installed.
                           [01]: KB5003254
                           [02]: KB4562830
                           [03]: KB4577586
                           [04]: KB4580325
                           [05]: KB4586864
                           [06]: KB4589212
                           [07]: KB4593175
                           [08]: KB4598481
                           [09]: KB5003637
                           [10]: KB5003503
Network Card(s):           11 NIC(s) Installed.
                           [01]: VMware Virtual Ethernet Adapter for VMnet0
                                 Connection Name: VMware Network Adapter VMnet0
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 192.168.0.1
                                 [02]: fe80::2811:45e8:afa5:ffbd
                           [02]: VMware Virtual Ethernet Adapter for VMnet8
                                 Connection Name: VMware Network Adapter VMnet8
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 192.168.1.1
                                 [02]: fe80::9081:59d1:cd64:4ee5
                           [03]: VMware Virtual Ethernet Adapter for VMnet1
                                 Connection Name: VMware Network Adapter VMnet1
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 192.168.88.1
                                 [02]: fe80::fc7d:bd5a:c57d:7752
                           [04]: Windscribe VPN
                                 Connection Name: Ethernet 2
                                 Status:          Media disconnected
                           [05]: Intel(R) Ethernet Connection (7) I219-V
                                 Connection Name: Ethernet
                                 DHCP Enabled:    Yes
                                 DHCP Server:     192.168.1.1
                                 IP address(es)
                                 [01]: 192.168.1.37
                                 [02]: fe80::fdc4:fb7f:10b1:4bc
                           [06]: Bluetooth Device (Personal Area Network)
                                 Connection Name: Connexion réseau Bluetooth 2
                                 Status:          Media disconnected
                           [07]: TAP-ProtonVPN Windows Adapter V9
                                 Connection Name: Connexion au réseau local
                                 Status:          Media disconnected
                           [08]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (Default Switch)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 172.24.112.1
                                 [02]: fe80::acb2:b97e:2713:a244
                           [09]: VirtualBox Host-Only Ethernet Adapter
                                 Connection Name: VirtualBox Host-Only Network
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 192.168.56.1
                                 [02]: fe80::d55b:3892:2a51:a66e
                           [10]: VirtualBox Host-Only Ethernet Adapter
                                 Connection Name: VirtualBox Host-Only Network #2
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 192.168.99.1
                                 [02]: fe80::bddf:990d:9b90:2116
                           [11]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (WSL)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 172.20.32.1
                                 [02]: fe80::a953:655:d5b4:78b
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.
```

### Steps to reproduce

  1. download crc binaries and move to right folder
  2. run "crc setup" (successful)
  3. run "crc start" (fail)
 

### Expected
crc start succeed with VM started

### Actual
crc start fails with error : 
```
Get "http://unix/version": open \\.\pipe\crc-admin-helper: Access is denied.
```

### Logs

(check gist link below)

Gist link of  `crc start --log-level debug`  :
https://gist.github.com/w4hf/80009bda52334a6bf8e778ba4f43765f

**Comment by guillaumerose** [comment]
Did you reboot your computer after the first `crc setup` ? It should have asked you to do it. `crc` creates a group and add the current user in it. 
You need to logout/login again to really be in the group.

**Comment by w4hf** [comment]
> 
> 
> Did you reboot your computer after the first `crc setup` ? It should have asked you to do it. `crc` creates a group and adds the current user in it.
> You need to logout/login again to really be in the group.

Bingo!
The thing is the ```crc setup``` command got stuck in ```adding user to crc-users group``` (but this is another issue) so I added the user manually using lusrmgr.msc but forgot to reboot !
Thank you @guillaumerose 
Closing issue
